test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -103,6 +103,20 @@ public void dropAllObjects() throws SQLException {
     JdbcFixture.dropAllObjects();
   }
 
+  private String nowPlus(int minutes) {
+    switch (JdbcFixture.DATABASE) {
+      case H2:
+        return "dateadd('minute', " + minutes + ", now())";
```

Comment:
current_timestamp() is standard SQL.  It doesn't appear that SQL Server or Oracle support now(), for example, while I suspect all 4 support current_timestamp(). I know this is in a database-type switch, but I still think it's useful to avoid unnecessary non-portable constructs.